addons/block_code/blocks/communication/switch_scene.tres

```diff
@@ -0,0 +1,18 @@
+[gd_resource type="Resource" load_steps=2 format=3 uid="uid://dnc2555wnobks"]
+
+[ext_resource type="Script" path="res://addons/block_code/code_generation/block_definition.gd" id="1_rnqd5"]
+
+[resource]
+script = ExtResource("1_rnqd5")
+name = &"switch_scene"
+target_node_class = ""
+description = "Delete the current scene and switch to a new scene"
```

Comment:
How about "Delete the current scene and switch to the target scene"?

Reply:
"delete" seems the wrong word. You're not deleting anything, just switching.

Reply:
Perhaps something like:

> Stop playing the current scene, and switch to a different one. You might use this to switch to a new level.

Reply:
Simply place the Godot document [change_scene_to_file](https://docs.godotengine.org/en/stable/classes/class_scenetree.html#class-scenetree-method-change-scene-to-file)

Reply:
According to [change_scene_to_packed](https://docs.godotengine.org/en/stable/classes/class_scenetree.html#class-scenetree-method-change-scene-to-packed)'s description relating to `change_scene_to_file()`: `At the end of the frame, the formerly current scene, already removed from the tree, will be deleted (freed from memory) and then the new scene will be instantiated and added to the tree.`, the "delete" happens.  However, the "delete" might be less important for beginners.

Reply:
I would expect that using "delete" would be actively misleading for beginners. This is using `delete` in the "free()" sense, but in normal use when you "delete" something, it is gone forever.

Reply:
I stand by my suggestion above:

```suggestion
description = "Stop playing the current scene, and switch to a different one. You might use this to switch to a new level."
```